lib/msf/core/exploit/remote/http/http_cookie.rb

```diff
@@ -253,8 +253,13 @@ def valid_for_uri?(uri)
           end
 
           # Tests if it is OK to accept this cookie if it is sent from the passed +uri+.
+          #
+          # @param [String] uri The uri that will be checked
+          # @return [False] False will be returned if the nil or resolves to a blank string
+          # @return [True] True if is an acceptable URI
```

Comment:
`@return [Boolean] True if the URI is an acceptable URI, false if the URI is empty or resolves to a blank string`.

Multiple `@return` entries will confuse tools and in this case both True and False are subclasses of Boolean class.

Reply:
Note that if you truely want to denote multiple return values in YARD you can do this with the following syntax:

```
@return [Boolean, Array, String] Returns a boolean, array or string depending on XXX.
```

Reply:
Good catch, will get that updated 👍